packages/perf-test/tasks/perf-test.ts

```diff
@@ -191,18 +232,27 @@ function createScenarioTable(stories, testResults: CookResults, showAll: boolean
     resultsToDisplay
       .map(resultKey => {
         const testResult = testResults[resultKey]
-        const kind = getKindKey(resultKey)
-        const story = getStoryKey(resultKey)
-        const iterations = getIterations(stories, kind, story)
-        const tpi = getTpiResult(testResults, stories, kind, story) || 'n/a'
-        const fabricTpi = getTpiResult(testResults, stories, kind, 'Fabric') || ''
```

Comment:
I created `linkifyResults` to reduce duplication and then forgot to use it here. Thanks for fixing. 😄